integration_test/.env

```
@@ -24,12 +24,18 @@ SearchByTitle=titleUsedForSearching
-SSO_URL=ssoURL
 GroupID=groupID
 Receiver=receiverUser
 ReceiverPass=passOfReceiverUser
 
+# SSO_URL — OPTIONAL. Controls the authentication path:
+#   - Not set  → m.login.password (direct password login to homeserver)
+#   - Set      → OIDC bypass (HTTP replay against LemonLDAP SSO)
+#
+# Only set SSO_URL if your USERNAME/PASSWORD are valid on that SSO provider.
+# Staging accounts do NOT exist in Linagora LDAP — leave SSO_URL empty.
+SSO_URL= ssoURL
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)